demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -77,7 +77,7 @@ def __init__(
             validate_all=False, is_external_repo=False, skip_pack_rn_validation=False, print_ignored_errors=False,
             silence_init_prints=False, no_docker_checks=False, skip_dependencies=False, id_set_path=None, staged=False,
             create_id_set=False, json_file_path=None, skip_schema_check=False, debug_git=False, include_untracked=False,
-            pykwalify_logs=False, quite_bc=False,
+            pykwalify_logs=False, check_unskipped_playbooks=True, quite_bc=False
```

Comment:
since you are not exposing this flag externally please do not pass it as an argument to the init method.

Reply:
exposing externally now